src/coupling/balance.hh

```diff
@@ -277,7 +279,7 @@ public:
      * possibly done when we will have a boundary mesh.
 	 */
 	void add_flux_matrix_values(unsigned int quantity_idx,
-			unsigned int boundary_idx,
+			SideIter side,
```

Comment:
Would be better to pass DHCellSide which also know the dofs, so we can avoid passing them explicitly in separate parameter.